plugins/build.gradle

```diff
@@ -0,0 +1,5 @@
+sonarqube {
```

Comment:
This subproject is not required IMO.

Reply:
Required to skip SonarQube analyses on this project and sub projects